cmd/celestia-appd/cmd/addr_converter.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+)
+
+// AddrConversionCmd returns a command that converts between celestia1xxx and
+// celestiavaloper1xxx addresses.
+func AddrConversionCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "addr-conversion [celestia address]",
+		Short: "Convert a celestia1xxx address to a validator operator address celestiavaloper1xxx",
```

Comment:
What about to the consensus address (celestiavalcons)? Is that necessary?

See https://docs.cosmos.network/main/build/spec/addresses/bech32#hrp-table

Reply:
I would like to think there's some existing tooling here we can import from the SDK then having to write it ourselves but maybe I'm wrong